src/backend/states/baseState.ts

```diff
@@ -263,6 +263,7 @@ export abstract class BaseState {
 
         return {
             __SPECTOR_Object_TAG: WebGlObjects.getWebGlObjectTag(object) || this.options.tagWebGlObject(object),
+            __SPECTOR_Object_Source: WebGlObjects.getWebGlObjectSource(object),
```

Comment:
I believe this is dead code now? If not, this should be `__SPECTOR_Object_Source` although I plan to rename it to `__SPECTOR_Object_ORIGIN` or `__SPECTOR_Object_LIFETIME` or something.